staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -1106,6 +1106,7 @@ func emitCallsToValidators(c *generator.Context, validations []validators.Functi
 
 	for _, v := range validations {
 		isFatal := v.Flags().IsSet(validators.IsFatal)
```

Comment:
with NonError in the mix, I find IsFatal a bit difficult to grok.  Does "short circuit"  or some term along those lines resonate with you at all for this flags purpose?

Reply:
Yeah "Fatal" as a name is a bit dramatic now.  We could do "FatalStop" and "NonFatalStop" as distinct flags, but I don't love that.  They sort of feel orthogonal: 
Fatal=0, NonError=0 => error + continue validation
Fatal=1, NonError=0 => error + abort further validation
Fatal=0, NonError=1 => nonsensical
Fatal=1, NonError=1 => no error, but abort further validation

We could rename to ShortCircuit as you suggest, and leave that 1/4 case as "useless but consistent".  Will do.